Readme.md

```diff
@@ -1,39 +1,103 @@
-Regit docker images
-===================
+# Regit docker images
 
-Docker images used for setting up development and speeding up CI builds.
+Docker images used for setting up development and speeding up CI builds in the Regit project.
```

Comment:
....speeding up CI builds in Regit projects.

Reply:
Done